src/components/molecules/UserProfilePicture/UserProfilePicture.tsx

```diff
@@ -114,40 +114,40 @@ const UserProfilePicture: React.FC<UserProfilePictureProp> = ({
           style={{ ...avatarStyle }}
         />
 
-        {Reactions.map(
-          (reaction, index) =>
-            reactions.find(
-              (r) => r.created_by === user.id && r.reaction === reaction.type
-            ) && (
-              <div key={index} className="reaction-container">
-                <S.Reaction
-                  role="img"
-                  aria-label={reaction.ariaLabel}
-                  className={reaction.name}
-                  reactionPosition={reactionPosition}
-                >
-                  {reaction.text}
-                </S.Reaction>
-
-                {!muteReactions && !isAudioEffectDisabled && (
-                  <audio autoPlay loop>
-                    <source src={reaction.audioPath} />
-                  </audio>
-                )}
-              </div>
-            )
-        )}
-        {messagesToBand && (
-          <div className="reaction-container">
+        <div className="reaction-container">
+          {Reactions.map(
+            (reaction, index) =>
+              reactions.find(
+                (r) => r.created_by === user.id && r.reaction === reaction.type
+              ) && (
+                <>
```

Comment:
@mike-lvov FYI: This should have used the named `<Fragment>` component, which accepts a `key` prop:

- https://reactjs.org/docs/fragments.html#keyed-fragments